.github/workflows/sync-issues.yml

```diff
@@ -39,26 +36,29 @@ jobs:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
 
+      - name: Check Memory Usage Before
+        run: free -m
+
       - name: Sync Issues
         env:
+          NODE_OPTIONS: "--max-old-space-size=14336" # 14 GB
```

Comment:
It seems wrong to need 14GB here. What is it for?

Reply:
I'm removing it now. I realized I was committing hundreds of times in a loop and just made it a single commit after the loop.

Reply:
It works.